backend/app/services/pdf/templates/resume.css

```
@@ -39,13 +40,17 @@ h2 {
   border-bottom: 2px solid #333;
   padding-bottom: 1mm;
   margin: 6mm 0 3mm 0;
+  /* 見出しがページ末尾に孤立して直後の本文だけ次ページに行くのを防ぐ */
+  page-break-after: avoid;
```

Stylelint (17.12.0)

[error] Expected "page-break-after" to be "break-after" (property-no-deprecated)

(property-no-deprecated)